datasets/wino_bias/wino_bias.py

```diff
@@ -0,0 +1,312 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""WinoBias: Winograd-schema dataset for detecting gender bias"""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import datasets
+
+
+_CITATION = """\
+@article{DBLP:journals/corr/abs-1804-06876,
+  author    = {Jieyu Zhao and
+               Tianlu Wang and
+               Mark Yatskar and
+               Vicente Ordonez and
+               Kai{-}Wei Chang},
+  title     = {Gender Bias in Coreference Resolution: Evaluation and Debiasing Methods},
+  journal   = {CoRR},
+  volume    = {abs/1804.06876},
+  year      = {2018},
+  url       = {http://arxiv.org/abs/1804.06876},
+  archivePrefix = {arXiv},
+  eprint    = {1804.06876},
+  timestamp = {Mon, 13 Aug 2018 16:47:01 +0200},
+  biburl    = {https://dblp.org/rec/journals/corr/abs-1804-06876.bib},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+"""
+
+_DESCRIPTION = """\
+WinoBias, a Winograd-schema dataset for coreference resolution focused on gender bias.
+The corpus contains Winograd-schema style sentences with entities corresponding to people
+referred by their occupation (e.g. the nurse, the doctor, the carpenter).
+"""
+
+_HOMEPAGE = "https://uclanlp.github.io/corefBias/overview"
+
+_LICENSE = "MIT License (https://github.com/uclanlp/corefBias/blob/master/LICENSE)"
+
+_URL = "https://drive.google.com/uc?export=download&confirm=yLNb&id=14Im3BnNl-d2fYETYmiH5yq6eFGLVC3g0"
+
+
+class WinoBias(datasets.GeneratorBasedBuilder):
+    """WinoBias: Winograd-schema dataset for detecting gender bias"""
+
+    VERSION = datasets.Version("4.0.0")
+
+    # This is an example of a dataset with multiple configurations.
+    # If you don't want/need to define several sub-sets in your dataset,
+    # just remove the BUILDER_CONFIG_CLASS and the BUILDER_CONFIGS attributes.
+
+    # If you need to make complex sub-parts in the datasets with configurable options
+    # You can create your own builder configuration class to store attribute, inheriting from datasets.BuilderConfig
+    # BUILDER_CONFIG_CLASS = MyBuilderConfig
+
+    # You will be able to load one or the other configurations in the following list with
+    # data = datasets.load_dataset('my_dataset', 'first_domain')
+    # data = datasets.load_dataset('my_dataset', 'second_domain')
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="wino_bias",
+            version=VERSION,
+            description="WinoBias: Winograd-schema dataset for detecting gender bias",
+        ),
+    ]
+
+    @property
+    def manual_download_instructions(self):
+        return """\
+             You need to manually download the 'anonymized.augmented.train.english.v4_auto_conll' file from Google Drive
+             (https://drive.google.com/uc?export=download&confirm=yLNb&id=14Im3BnNl-d2fYETYmiH5yq6eFGLVC3g0). The folder containing the saved file
```

Comment:
You don't have to use manual data here

You can pass the url `https://drive.google.com/uc?export=download&id=14Im3BnNl-d2fYETYmiH5yq6eFGLVC3g0` directly to the `dl_manager` and it will download the file.

Reply:
Hi, @lhoestq , I did try the download_manager, but unfortunately it was not working. It points to the page of the download rather than the link to download the file itself. After discussion with @yjernite I re-wrote to download manually. 
Even now when I use the link provided by you, it still points to the download page itself.

Reply:
The url should work with the download manager

For example for me when I run 
```python
from datasets import DownloadManager

DownloadManager().download("https://drive.google.com/uc?export=download&id=14Im3BnNl-d2fYETYmiH5yq6eFGLVC3g0")
```

it downloads the file.

What issue do you have with this link ?

Reply:
Hi @lhoestq, thank you for persisting with me. When I try the code given by you (as is), it does download the file. But when I implement it in my code, I get the following error.

`>>> data = load_dataset('./datasets/wino_bias')
Downloading and preparing dataset wino_bias/wino_bias (download: Unknown size, generated: 82.91 MiB, post-processed: Unknown size, total: 82.91 MiB) to C:\Users\aksha\.cache\huggingface\datasets\wino_bias\wino_bias\4.0.0\81b712b8f7b261b5f1239cd953aa3dc38716fc03b99ff295acc50c12bda231c0...
Downloading: 269MB [01:01, 4.40MB/s] 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "d:\personal\huggingface\datasets\src\datasets\load.py", line 610, in load_dataset
    ignore_verifications=ignore_verifications,
  File "d:\personal\huggingface\datasets\src\datasets\builder.py", line 527, in download_and_prepare
    dl_manager=dl_manager, verify_infos=verify_infos, **download_and_prepare_kwargs
  File "d:\personal\huggingface\datasets\src\datasets\builder.py", line 587, in _download_and_prepare
    self.info.download_checksums, dl_manager.get_recorded_sizes_checksums(), "dataset source files"
  File "d:\personal\huggingface\datasets\src\datasets\utils\info_utils.py", line 34, in verify_checksums
    raise UnexpectedDownloadedFile(str(set(recorded_checksums) - set(expected_checksums)))
datasets.utils.info_utils.UnexpectedDownloadedFile: {'https://drive.google.com/uc?export=download&id=14Im3BnNl-d2fYETYmiH5yq6eFGLVC3g0'}`

I thought it is because of bad cache or me trying to redownload the data, but it persists even after clearing the cache. Coincidentally it always fails after downloading about 269 MBs of data while the original file downloaded manually is about 256 MBs.